pkg/certs/cert-inspection/certgraphanalysis/nameguessing.go

```diff
@@ -46,7 +46,7 @@ var secretLocationToLogicalName = map[certgraphapi.InClusterSecretLocation]logic
 	newSecretLocation("openshift-kube-apiserver", "service-network-serving-certkey"):                  newMeaning("kube-apiserver-service-network-serving", "Serving certificate used by the kube-apiserver to terminate requests via the service network."),
 	newSecretLocation("openshift-kube-apiserver-operator", "localhost-serving-signer"):                newMeaning("kube-apiserver-localhost-signer", "Signer used by the kube-apiserver to create serving certificates for the kube-apiserver via localhost."),
 	newSecretLocation("openshift-kube-apiserver", "localhost-serving-cert-certkey"):                   newMeaning("kube-apiserver-localhost-serving", "Serving certificate used by the kube-apiserver to terminate requests via localhost."),
-	newSecretLocation("openshift-machine-config-operator", "machine-config-server-tls"):               newMeaning("mco-mystery-cert", "TODO: team needs to make description"),
+	newSecretLocation("openshift-machine-config-operator", "machine-config-server-tls"):               newMeaning("mco-mcs-cert", "Serving certificate used by machine config server to serve Ignition to nodes"),
```

Comment:
The server is contacted to scale up new nodes.

Reply:
added "during node scaling."